app/policies/application_policy.rb

```diff
@@ -63,5 +63,10 @@ def admin_only
     !!user&.admin?
   end
 
+  def super_user_only
+    admin_only && user.admin_profile.super_user?
```

Comment:
I thought you had a bug here because the `admin_only` suggests the user should have only one profile, but instead it's checking if the user is also an admin.

I think it could be simplified to
```suggestion
    !!user.admin_profile&.super_user?
```

Reply:
I think I did it the way I did because it makes it clear it's like the admin role **plus** another level (super user).